network/p2p/node/resourceManager_test.go

```diff
@@ -260,6 +260,12 @@ func TestCreateStream_ProtocolLimitLessThanPeerProtocolLimit(t *testing.T) {
 }
 
 func TestCreateStream_ProtocolLimitGreaterThanPeerProtocolLimit(t *testing.T) {
+	// TODO: with libp2p upgrade to v0.32.2; this test is failing as the peer protocol limit is not being enforced, and
+	// rather the protocol limit is being enforced, this test expects each peer not to be allowed more than 5 streams on a specific protocol.
+	// However, the maximum number of streams on a specific protocol (and specific protocol) are being enforced instead.
```

Comment:
did you mean peer here?
```suggestion
	// However, the maximum number of streams on a specific protocol (and specific peer) are being enforced instead.
```